apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -16,6 +17,27 @@ export const RowType = {
   STUDENT: "studentRow",
 };
 
+// Constants around moving students to another section.
+// OTHER_TEACHER - value determines whether students will be moved to a section owned by a different teacher
+// COPY_STUDENTS - value determines whether students will be copied to the new section or moved (and subsequently removed from current section)
+export const OTHER_TEACHER = "otherTeacher";
+export const COPY_STUDENTS = "copy";
```

Comment:
Also, it would be more consistent with our style to write the comment closer to each constant, e.g.:
```
// Constants around moving students to another section.

// whether students will be moved to a section owned by a different teacher
export const OTHER_TEACHER = "otherTeacher";

// ...
export const COPY_STUDENTS = "copyStudents";
```